### PR TITLE
Profile model

### PR DESCRIPTION
**Pull request by Abhi-S-1**
### What this PR does (required):
- Created Model for Profile 

### Any information needed to test this feature (required):
- View profile model

### CODE REVIEW

File: server/models/Availability.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const mongoose = require("mongoose");
+
+const availabilitySchema = new mongoose.Schema({
+  week_of: {
```
Comment (moffatethan) [inline server/models/Availability.js]: `weekOf` let's preference camelCase please as it's JavaScript convention.

File: server/models/Availability.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    type: Date,
+    required: true,
+  },
+  monday: {
```
Comment (moffatethan) [inline server/models/Availability.js:8]: Not quite the approach for availability I want to take but we'll work more on it this week.

File: server/models/Profile.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+const mongoose = require("mongoose");
+const { Availability } = require("./Availability");
+
+const profileSchema = new mongoose.Schema({
+  first_name: {
```
Comment (moffatethan) [inline server/models/Profile.js]:
```suggestion
  firstName: {
```

Ditto throughout the schema 🚀

File: server/models/Profile.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  },
+  address: {
+    type: String,
+  },
+  description: {
+    type: String,
+  },
+  pet_sitter: {
+    type: Boolean,
+  },
```
Comment (moffatethan) [inline server/models/Profile.js]: Nice 👍🏻

**Review by moffatethan — CHANGES_REQUESTED** [review]
Just some changes regarding code style. Rest looks fine to me!

**Comment by Abhi-S-1** [comment]
Thanks for the comments. Changes Made 👍🏻

**Review by moffatethan — APPROVED** [review]

**Pull request merged**